js/script.js

```diff
@@ -13,20 +13,39 @@
  **/
 
 function Generator() {
-
     /* Name Arrays: Customize names to change possible output */
+  this.initialize = function () {
+    document.getElementById("enter").addEventListener("click", this.createRapName);
+  }
+
+  this.createRapName = function () {
+    this.userName = document.getElementById("user-input").value.toString();
```

Comment:
You can do this with jquery or with plain javascript. I would recommend picking one and sticking to it.